app/models/profile.rb

```diff
@@ -17,6 +17,8 @@ class Profile < ApplicationRecord
     verification_cancelled: 4,
     in_person_verification_pending: 5,
     threatmetrix_review_pending: 6,
+    in_person_verification_deactivated: 7,
+    in_person_verification_pending_and_deactivated: 8,
```

Comment:
Since we can deactivate profiles that are either pending or passed we need two deactivation statuses, so that we can restore the profile to the correct original status when we reactivate